apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/deployments/[deploymentSlug]/DeploymentNavBar.tsx

```diff
@@ -51,6 +52,11 @@ export const DeploymentsNavBar: React.FC = () => {
             </NavigationMenuItem>
           </NavigationMenuList>
         </NavigationMenu>
+        <div className="flex-grow items-center">
```

Comment:
the div is wrapping a component that conditionally renders, this mean sometimes we just have an empty div. is that what we want?

Reply:
The system will generally never be empty? I think this is only needed to make TS happy.

Reply:
We can make it reroute if the system is null to the systems page if that is better.